code/game/objects/items.dm

```diff
@@ -127,6 +127,12 @@
 	if(ismob(loc))
 		dropped(loc)
 
+	for(var/atom/movable/A in contents_recursive()) // Get rid of any intel objects so we don't delete them
+		if(isitem(A))
```

Comment:
instead of doing a typecheck, just for `for(var/obj/item/item in contents_recursive())`, that makes it automatically make a typecheck